### DISCUSSION TITLE
prevent animals from being sold automatically

### DISCUSSION THREAD

**Discussion by 29EUSEBIO29** [Fetty42/FS22_RealisticAnimalLosses#1] [General]
good I really liked your mod, I would like to know if it is possible for animals not to be sold or die for dedicated servers. As I have a dedicated server, the game is always on 24 hours a day

**Comment by Fetty42** [comment]
Hi,
I don't understand exactly what you mean. Do you have a problem with animals dying in stables that do not currently belong to anyone? Currently, the server job only looks at these stables/animals.
